backend/python/tapa/core.py

```diff
@@ -22,9 +22,11 @@
 from tapa.floorplan import get_floorplan_result, generate_floorplan, checkpoint_floorplan
 from tapa.verilog import ast
 from tapa.verilog import xilinx as rtl
+from tapa.hardware import get_slr_num
```

Comment:
Prefer "count" over "num" when referring to the total number of items. "Number" can be interpreted as "index" in many cases.